RELEASE.md

```diff
@@ -62,12 +62,14 @@ Wait for the evergreen version to finish, and ensure that the release task compl
 Check that the released files, library and symbols, are available at the following URLs:
 - Windows
   - Release build
-    - `https://translators-connectors-releases.s3.us-east-1.amazonaws.com/mongosql-odbc-driver/windows/${release_version}/release/mongoodbc.dll`  
-    - `https://translators-connectors-releases.s3.us-east-1.amazonaws.com/mongosql-odbc-driver/windows/${release_version}/release/mongoodbc.pdb`
+    - `https://translators-connectors-releases.s3.us-east-1.amazonaws.com/mongosql-odbc-driver/windows/${release_version}/release/atsql.dll`
+    - `https://translators-connectors-releases.s3.us-east-1.amazonaws.com/mongosql-odbc-driver/windows/${release_version}/release/atsqls.dll`
+    - `https://translators-connectors-releases.s3.us-east-1.amazonaws.com/mongosql-odbc-driver/windows/${release_version}/release/atsql.pdb`
     - `https://translators-connectors-releases.s3.us-east-1.amazonaws.com/mongosql-odbc-driver/windows/${release_version}/release/mongoodbc.msi`
   - Debug build with logging capabilities
```

Comment:
I think that now the debug build does not have different capabilities. Both release and debug can log. Maybe can remove this section entirely. The important artifact being able to get the pdb if needed to investigate crashes and we have it with the release build.